### PR TITLE
feat(ldp): subscriptions list implementation

### PR DESCRIPTION
**Pull request by vovh**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/logs-to-customer-module`  
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-15919
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

### CODE REVIEW

**Review by pauldkn — APPROVED** [review]

File: packages/manager/modules/logs-to-customer/src/components/subscriptions/SubscriptionEmpty.component.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import { useTranslation } from 'react-i18next';
+import { useNavigate } from 'react-router-dom';
+
+const SubscriptionEmpty = () => {
+  const navigate = useNavigate();
+  const { t } = useTranslation('logSubscription');
+
+  return (
+    <OsdsTile rounded inline className="flex flex-col w-full h-fit">
+      <div className="flex flex-col gap-6 pb-4">
```
Comment (rjamet-ovh) [inline packages/manager/modules/logs-to-customer/src/components/subscriptions/SubscriptionEmpty.component.tsx:21]:
```suggestion
      <div className="flex flex-col gap-6">
```
padding is not needed

File: packages/manager/modules/logs-to-customer/src/components/subscriptions/SubscriptionStreamActions.component.tsx [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+  const { t: tSubscription } = useTranslation('logSubscription');
+  const { data, isLoading, isPending, error } = useLogStreamUrl(
+    subscription.serviceName,
+    subscription.streamId,
+  );
+  const [streamUrl, setStreamUrl] = useState<Url | null>(null);
+
+  useEffect(() => {
+    setStreamUrl(
+      data?.data.find(({ type }) => type === UrlTypeEnum.GRAYLOG_WEBUI),
```
Comment (rjamet-ovh) [inline packages/manager/modules/logs-to-customer/src/components/subscriptions/SubscriptionStreamActions.component.tsx:42]: Best to use the `select` option directly on the useQuery in `useLogStreamUrl`

File: packages/manager/modules/logs-to-customer/src/components/subscriptions/SubscriptionTile.component.tsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+type SubscriptionTileProps = {
+  subscription: LogSubscription;
+};
+
+const SubscriptionTile = ({ subscription }: SubscriptionTileProps) => {
+  const { t } = useTranslation('logSubscription');
+
+  return (
+    <OsdsTile rounded inline className="flex flex-col w-full h-fit">
+      <div className="flex flex-col gap-6 pb-4">
```
Comment (rjamet-ovh) [inline packages/manager/modules/logs-to-customer/src/components/subscriptions/SubscriptionTile.component.tsx:20]:
```suggestion
      <div className="flex flex-col gap-6">
```

File: packages/manager/modules/logs-to-customer/src/data/api/logService.ts [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import apiClient from '@ovh-ux/manager-core-api';
+import { Service } from '../types/dbaas/logs';
+
+/**
+ * GET log service infos
+ */
+export const getLogServicev6 = async (serviceName: string) =>
```
Comment (rjamet-ovh) [inline packages/manager/modules/logs-to-customer/src/data/api/logService.ts:7]: As there's no need to handle multiple API version, you can remove the suffix
```suggestion
export const getLogService = async (serviceName: string) =>
```

File: packages/manager/modules/logs-to-customer/src/data/mocks/logSubscription.mock.ts [hunk 5]
```diff
@@ -0,0 +1,22 @@
+import { LogSubscription, LogSubscriptionResource } from '../types/dbaas/logs';
+
+export const logSubscriptionsMock: LogSubscription[] = [
+  {
+    serviceName: 'SubscriptionMock',
+    createdAt: '2024-10-23T15:24:23Z',
+    kind: 'B5F995F2-BE9C-4805-8910-9A4E5DD25EBF',
```
Comment (rjamet-ovh) [inline packages/manager/modules/logs-to-customer/src/data/mocks/logSubscription.mock.ts:7]:
```suggestion
    kind: 'kind-name',
```

**Review by rjamet-ovh — APPROVED** [review]

**Pull request merged**